ibm_catalog.json

```diff
@@ -0,0 +1,183 @@
+              {
+                "diagram": {
+                  "caption": "Creates IBM Container Registry namespace.",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-container-registry/main/reference-architecture/deployable-architecture-icr.svg",
+                  "type": "image/svg+xml"
+                },
+                "description": "This architecture creates IBM Container Registry namespace and provides the ability to configure pull traffic limits and storage quotas, both in megabytes. It ensures efficient management of container image access by regulating the volume of data that can be pulled from the registry and by setting specific storage capacity limits."
+              }
+            ]
+          },
+          "configuration": [
+              {
+                "key": "ibmcloud_api_key"
+              },
+              {
+                "key": "use_existing_resource_group"
+              },
+              {
+                "key": "resource_group_name"
+              },
+              {
+                "key": "region",
+                "default_value": "us-south",
+                "options": [
+                  {
+                    "displayname": "Tokyo (jp-tok)",
+                    "value": "jp-tok"
+                  },
+                  {
+                    "displayname": "Sydney (au-syd)",
+                    "value": "au-syd"
+                  },
+                  {
+                    "displayname": "Sao Paulo (br-sao)",
+                    "value": "br-sao"
+                  },
+                  {
+                    "displayname": "Toronto (ca-tor)",
+                    "value": "ca-tor"
+                  },
+                  {
+                    "displayname": "Frankfurt (eu-de)",
+                    "value": "eu-de"
+                  },
+                  {
+                    "displayname": "Madrid (eu-es)",
+                    "value": "eu-es"
+                  },
+                  {
+                    "displayname": "London (eu-gb)",
+                    "value": "eu-gb"
+                  },
+                  {
+                    "displayname": "Global (global)",
+                    "value": "global"
+                  },
+                  {
+                    "displayname": "Osaka (jp-osa)",
+                    "value": "jp-osa"
+                  },
+                  {
+                    "displayname": "Dallas (us-south)",
+                    "value": "us-south"
+                  }
+                ]
+              },
+              {
+                "key": "name"
+              },
+              {
+                "key": "tags"
+              },
+              {
+                "key": "images_per_repo"
+              },
+              {
+                "key": "retain_untagged"
+              },
+              {
+                "key": "container_registry_endpoint",
+                "default_value": "us.icr.io"
```

Comment:
don't add default values in this file. If you leave it out, it will get pulled from the terraform default value. Otherwise you would need to maintain in 2 places

Reply:
addressed